src/lib/structures/var.ml

```diff
@@ -59,6 +59,10 @@ let equal a b = compare a b = 0
 
 let hash { id; _ } = id
 
+let underscore =
+  Random.self_init ();
+  of_string @@ Format.sprintf "_%d" (Random.int 1_000_000)
```

Comment:
Wait, I was not aware we were doing that — this does not seem safe at all! We should use a counter instead (there are no guarantees that two consecutive `Random.int` calls won't return the same value, which would be problematic as `forall x, y : int. x + y` would become `forall _1, _1: int. _1 + _1` which… is not the same at all.

Reply:
Opened #977

Reply:
ah ah :) you don´t  the best part of our mighty codebase I see :) 
This snippet is pure gold. I shared it one year ago on zulip, best joke of my short software developer career.

Reply:
Ah! But what I said above is not true — underscore is a global variable, not a function. OK, that makes things better — there can "just" be conflicts with a variable named `_n` for some `n` that changes each time Alt-Ergo is run. We should just call this `"._"` instead, no?